includes/razorpay-webhook.php

```diff
@@ -54,28 +54,45 @@ public function process()
                     write_log(['message' => $e->getMessage()]);
                     return;
                 }
-            }
 
-            switch ($data['event'])
-            {
-                case 'payment.authorized':
-                    return $this->paymentAuthorized($data);
+                switch ($data['event'])
```

Comment:
@mayankamencherla Please do not push multiple things in a single PR.
The fix should have gone live independent of the feature.